README.md

```diff
@@ -1,3 +1,7 @@
 # Crushtool
 
 [![Build Status](https://travis-ci.org/cholcombe973/crushtool.svg?branch=master)](https://travis-ci.org/cholcombe973/crushtool)[![Coverage Status](https://coveralls.io/repos/github/cholcombe973/crushtool/badge.svg?branch=master)](https://coveralls.io/github/cholcombe973/crushtool?branch=master)
+
+High on a rocky promontory sat an Electric Monk on a bored horse. From under its rough woven cowl the Monk gazed unblinkingly down into another valley, with which it was having a problem.
+
+https://docs.rs/crushtool/badge.svg
```

Comment:
This helps but it needs to be in markdown format.  Something like:
[![Docs](https://docs.rs/crushtool/badge.svg)](https://docs.rs/crushtool)